politeiawww/www.go

```diff
@@ -628,6 +629,33 @@ func _main() error {
 			}
 		}
 
+	case userDBMysql:
+		// If old encryption key is set it means that we need
+		// to open a db connection using the old key and then
+		// rotate keys.
+		var encryptionKey string
```

Comment:
The codepath for rotating encryption keys should be the same for both cockroachdb and mysql.

Reply:
Hrmm not sure how to achieve that, `user.Database` doesn't export `RotateKeys` - note that we use `db.RotateKeys` and not `userDB.RotateKeys`.

Reply:
Let's add it to the `user.Database` interface then so that we don't duplicate code.  You can just leave an empty stub for the `localdb` implementation.  `localdb` doesn't support encrypting data at rest right now.

Reply:
Sweet, will do.